packages/components/slot.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { connect, useFills } from "frontity";
+import { Connect, Package } from "frontity/types";
+
+type SlotType = React.FC<Connect<Package, { name: string; data?: any }>>;
+
+/**
+ * A React component that renders a Fill
+ */
+const Slot: SlotType = ({ name, state, children, ...slotProps }) => {
+  // Get the data, either from props or the current link.
+  const data = slotProps.data || state.source.get(state.router.link);
+
+  // Get the fills for this name.
+  const fills = useFills(name);
+
+  return (
+    // Returning a <Fragment/> because returning an array of components results in a typescript error
```

Comment:
This is the issue that seems to appears otherwise:

<img width="770" alt="Screenshot 2020-06-08 at 18 06 53" src="https://user-images.githubusercontent.com/5417266/84089480-ad01b880-a9b4-11ea-9ec8-17b24f30a378.png">